lib/alphabetical-list-items.js

```diff
@@ -22,9 +20,7 @@ function alphaCheck(ast, file, language, done) {
 
     items.forEach((item) => {
       if (item.children.length) {
-        const lineStartOffset = item.children[0].children[0].position.start.offset;
-        const lineEndOffset = item.children[0].children[item.children[0].children.length - 1].position.end.offset;
-        const text = normalize(remark.process(contents.slice(lineStartOffset, lineEndOffset)));
+        const text = normalize(toString(item));
```

Comment:
Oh that’s an interesting case, I was thinking that it would be better to sort it by indeed preferring `a a, a z`.

I like your solution of using `item.children[0].children[0]`, but it might pose fragile for cases like empty list-items, if you’re adding tests I suggest inserting empty list-items, and list-items with just a code block too!